woocommerce/woocommerce-gutenberg-products-block
# Featured Category Block

Implements the featured category block. This is heavily based on the featured products block.

Fixes #591



#### Accessibility




### Screenshots

This is the edit view:
![edit](https://user-images.githubusercontent.com/90977/60589282-074fcd80-9d91-11e9-98e6-d76e6a443742.png)

This is how it looks in editor:
![editor](https://user-images.githubusercontent.com/90977/60589286-0ae35480-9d91-11e9-9775-3c57926a0b99.png)

This is the frontend view which matches the editor:
![frontend](https://user-images.githubusercontent.com/90977/60589289-0d45ae80-9d91-11e9-9c80-121fb69cf9d5.png)

### How to test the changes in this Pull Request:

1. Build
2. Add "Featured Category" block to the editor
3. Select a category. Ensure only one can be selected
4. Save. You'll see category content!
5. View on frontend
6. Repeat but this time try to set an image manually in the sidebar options. View and confirm it works as expected.



### Changelog

> New Block: Featured Category used to feature a link to a single product category.
